nncf/torch/quantization/layers.py

```diff
@@ -743,9 +743,15 @@ def get_parameters_for_torch_fq(self) -> Tuple[int, int, torch.Tensor, torch.Ten
 
             scale, zero_point = get_scale_zp_from_input_low_input_high(level_low, level_high, input_low, input_high)
 
+            if self.narrow_range:
+                if level_low < 0:
+                    level_low -= 1
+                else:
+                    level_high += 1
```

Comment:
Can we align it with?
https://github.com/openvinotoolkit/nncf/blob/68c7d39815927eee7cd5334ffb83b70f3eeaff0d/nncf/common/quantization/quantizers.py#L39-L43

Reply:
It's backward operation to get level_low and level_hight without narrow_range to get levels (255,256) for FQ that can be converted to ONNX